.redocly.yaml

```diff
@@ -1,20 +1,23 @@
 # See https://redoc.ly/docs/cli/configuration/ for more information.
-apiDefinitions:
-  main: openapi/openapi.yaml
+apis:
+  main@v1: 
```

Comment:
There is an issue in that adding a version to here (or changing the API name) breaks the registry build because the registry is doing something like `openapi lint main`. I could add a new API to the registry. 

Thoughts?

Reply:
This is expected. If you change the version of API name you have to adjust the source accordingly.

Reply:
But I cannot adjust the source.

<img width="771" alt="2022-03-23_07-06-50" src="https://user-images.githubusercontent.com/1161871/159695500-bf2cc8b7-5f27-4bdb-b936-c844b512fbb5.png">

Maybe I could adjust it after merging?

Reply:
Ah, I've got it. Yes, after merging.